src/main/resources/application.yml

```diff
@@ -9,6 +9,11 @@ spring:
   main:
     allow-bean-definition-overriding: true
 
+springdoc:
```

Comment:
Ah - magi :)

Reply:
https://springdoc.org/#selecting-the-rest-controllers-to-include-in-the-documentation